collection/collection/src/commonMain/kotlin/androidx/collection/SimpleArrayMap.kt

```diff
@@ -288,7 +289,8 @@ public open class SimpleArrayMap<K, V> @JvmOverloads public constructor(capacity
     // Java's Map interface, which is necessary since ArrayMap is written in Java and implements
     // both Map and SimpleArrayMap.
     public open fun getOrDefault(key: Any?, defaultValue: V): V {
-        return getOrDefaultInternal(key, defaultValue)
+        // TODO: revert the change: this function was changed in JB fork because of https://youtrack.jetbrains.com/issue/KT-65061
```

Comment:
Explicit parameters doesn't sound as bad thing that requires reverting

Reply:
In case of the merge conflict the change might be lost. I dont' want it to be lost until we don't need it anymore. 

And we can't upstream this change, because in common code the explicit parameter is not needed, it's redundant.